src/server/common/helpers/logging/logger-options.js

```diff
@@ -1,16 +1,17 @@
 import ecsFormat from '@elastic/ecs-pino-format'
 
 import { config } from '~/src/config'
+import { redactedUploadContext } from '~/src/server/common/helpers/logging/logging-context-upload-redaction'
 
 const isDevelopment = config.get('isDevelopment')
-const redactionPaths = [
+let redactionPaths = [
   'req.headers.authorization',
   'req.headers.cookie',
   'res.headers'
 ]
 
 if (!isDevelopment) {
-  redactionPaths.push('uploadDetails', 'uploadId')
+  redactionPaths = redactionPaths.concat(redactedUploadContext())
```

Comment:
If `redactedUploadContext` becomes an array (see below). And note `.push` can also take an unlimited amount of params. You can do:

```
redactionPaths.push(...redactedUploadContext)
```

Reply:
ah, I was trying to find the better way to do this :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push#syntax